djstripe/models.py

```diff
@@ -160,28 +141,11 @@ def process(self):
             "ping"
         """
         if self.valid and not self.processed:
+            kind_parts = self.kind.split(".")
+            type, sub_type = kind_parts[0], ".".join(kind_parts[1:])
```

Comment:
Change these to ``event_type`` and ``event_type`` to avoid reserved keyword collision.

Also, you can condense these two lines into one like so:

```python
event_type, event_subtype = self.kind.split(".", 1)
```

``split()`` takes an optional limiter that splits the string only that many times, which can be pretty useful.